client/src/lib.rs

```diff
@@ -205,6 +214,16 @@ pub enum GatewayInitError {
     TonicTransportError(#[from] tonic::transport::Error),
 }
 
+impl NamespacedServerGatewayOptions {
+    /// Attempt to establish a connection to the Temporal server
+    pub async fn connect(
+        &self,
+        _metrics_meter: Option<&Meter>,
+    ) -> Result<RetryGateway<NamespacedServerGateway>, GatewayInitError> {
+        todo!()
+    }
+}
+
 impl ServerGatewayOptions {
```

Comment:
Is namespace going to be removed from this struct?
What about the metrics? Will they be namespaceless?

Reply:
Ya, I just somehow missed that. It won't have it.

Metrics will always have namespaces (assuming the call involves a namespace). The only difference is at what point that label is bound. Namespaced clients would bind it when constructed, non-namespaced ones would bind it at call time.

Reply:
Wait, I didn't miss it. It is removed. https://github.com/temporalio/sdk-core/pull/263/files/3af8ae06394d089141f600f99ee09e42199b56a9#diff-3f7e738903cc8204a4e3b1c8ab4db5812898e6bb5ad8b6c7b36df42e29481a8fL75